CustomBlockManagerPlugin.inc.php

```diff
@@ -160,7 +160,7 @@ function manage($args, $request) {
 	 * @return boolean
 	 */
 	function isSitePlugin() {
-		return !Application::getRequest()->getContext();
+		return true;
```

Comment:
@bozana, won't make this plugin unusable by Managers who aren't also Admins?

Reply:
(Pinging @NateWr on this too, as this reverts his change, I think)

Reply:
Ah, @asmecher, I think you are right... Hmmm... So maybe then only not to consider this plugin in the upgrade script i.e. the other solution...